events/class_models/event_models.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+
+
+class Event(models.Model):
+    class Colors(models.TextChoices):
+        RED = "#FF0000", "Red"
+        PINK = "#FFC0CB", "Pink"
+        ORANGE = "#FFA500", "Orange"
+        YELLOW = "#FFFF00", "Yellow"
+        PURPLE = "#800080", "Purple"
+        GREEN = "#008000", "Green"
+        BLUE = "#0000FF", "Blue"
+        BROWN = "#A52A2A", "Brown"
+        WHITE = "#FFFFFF", "White"
+        GRAY = "#808080", "Gray"
+        BLACK = "#000000", "Black"
+
+    title = models.CharField(max_length=50)
+    location = models.CharField(blank=True, max_length=50)
+    description = models.TextField(blank=True)
+    date_time_start = models.DateTimeField()
+    date_time_end = models.DateTimeField()
```

Comment:
As @nemerna suggested in our demo,
I think that we can add default values to the title, date_time_start and date_time_end.
The default difference between the start and end of the event can be 30 minutes.

There are two ways you can implement it, 
1. set a default in the fields, (may cause some issues)
2. when validating if the field is left blank you can set the value of now and now + 30 minutes.

Reply:
Hi Tamir ,
1. IMHO title should be a a required field since it aint easy to find a collective name that can describe most of the meeting, 
2. i think default dates values and tests should be done under different PR

Reply:
Ok, start working on it. 😁

Reply:
on it :)